index.html

```diff
@@ -20,6 +20,12 @@
                 class="header__logo">
         </header>
         <main>
+            <div id='modal-container'>
+                <div class='modal'>
+                    <button class='modal-close'>Close</button>
+                    <div class="modal-content"></div>
+                </div>
+            </div>
```

Comment:
I see you chose to have this here. :))
I would prefer it on the other side as part of the instruction.
Let us add a `cursor: pointer;` on the close button

Reply:
@oxenprogrammer Do you mean having the "Close" button generated by the `showModal` function?